.github/workflows/usermods.yml

```
@@ -0,0 +1,71 @@
+          cache: 'pip'
+    - name: Install PlatformIO
+      run: pip install -r requirements.txt
+    - name: Add usermods environment
+      run: |
+        cp -v usermods/platformio_override.usermods.ini platformio_override.ini
+        echo >> platformio_override.ini
+        echo  "custom_usermods = ${{ matrix.usermod }}" >> platformio_override.ini
+        cat platformio_override.ini
+
+    - name: Build firmware
+      run: pio run -e ${{ matrix.environment }}    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)